app/scripts/templates/settings/account_recovery/recovery_key.mustache

```diff
@@ -12,8 +12,8 @@
 
         <div class="options">
             <div class="buttons">
-                <button class="print-key"><img alt="{{#t}}Print button image{{/t}}" class="recovery-icon-option" src="/images/recovery_key_print.svg">{{#t}}Print{{/t}}</button>
-                <button class="download-key"><img alt="{{#t}}Download button image{{/t}}" class="recovery-icon-option" src="/images/recovery_key_download.svg">{{#t}}Download{{/t}}</button>
+                <button class="print-key"><img alt="{{#t}}Print button image{{/t}}" class="recovery-icon-option print-image" src="">{{#t}}Print{{/t}}</button>
```

Comment:
Similar to avatar [image](https://github.com/mozilla/fxa-content-server/blob/0eab619612897dcda43e8074dafdf55e8cbe11ee/app/scripts/templates/settings/avatar_crop.mustache#L9) this uses `src=""` to avoid showing alt-text.